payment-settler/bin/payment-settler.js

```diff
@@ -0,0 +1,134 @@
+/** @import {MessageBatch} from 'cloudflare:workers' */
+import { getChainClient as defaultGetChainClient } from '../lib/chain.js'
+import filbeamAbi from '../lib/FilBeamOperator.abi.json'
+import { getDataSetsForSettlement } from '../lib/rail-settlement.js'
+import { TransactionMonitorWorkflow } from '@filbeam/workflows'
+import { handleTransactionRetryQueueMessage as defaultHandleTransactionRetryQueueMessage } from '../lib/queue-handlers.js'
+
+/**
+ * @typedef {{
+ *   type: 'transaction-retry'
+ *   transactionHash: string
+ * }} TransactionRetryMessage
+ */
+
+/**
+ * @typedef {{
+ *   ENVIRONMENT: 'dev' | 'calibration' | 'mainnet'
+ *   RPC_URL: string
+ *   FILBEAM_CONTRACT_ADDRESS: string
+ *   FILBEAM_CONTROLLER_ADDRESS_PRIVATE_KEY: string
```

Comment:
Cross-posting my question from https://github.com/filbeam/roadmap/issues/87#issuecomment-3421650448:

Do you remember if we use a regular private-key-based address or a Safe-based multisig as the entity allowed to settle CDN payment rails? If we use Safe, then I think we need to slightly change how the settlement works.

/cc @juliangruber